### PR TITLE
OptionsT: Undefined index: nested_parenthesis

### PR DESCRIPTION
**Pull request by arkener**
Reference: https://www.drupal.org/project/coder/issues/3117463

### CODE REVIEW

File: coder_sniffer/DrupalPractice/Test/General/OptionsTUnitTest.inc [hunk 1]
```diff
@@ -23,3 +23,12 @@ $form['display']['show_thumbnail'] = array(
...
 );
+
+$form['display']['status'] = [
+    '#title' => t('Status', array(), array('context' => 'test')),
+    '#type' => 'select',
+    '#options' => array(
+        '1' => 'Enabled',
+        '0' => t('Disabled', array(), array('context' => 'test')),
+    ),
+];
```
Comment (klausi) [inline coder_sniffer/DrupalPractice/Test/General/OptionsTUnitTest.inc]: there should be a newline at the end of the file.

File: coder_sniffer/DrupalPractice/Test/General/OptionsTUnitTest.inc [hunk 2]
```diff
@@ -23,3 +23,12 @@ $form['display']['show_thumbnail'] = array(
     '0' => t('No', array(), array('context' => 'test')),
   ),
 );
+
+$form['display']['status'] = [
+    '#title' => t('Status', array(), array('context' => 'test')),
```
Comment (klausi) [inline coder_sniffer/DrupalPractice/Test/General/OptionsTUnitTest.inc]: we should use 2 space indentation here as we do in the Drupal standard.

**Review by klausi — CHANGES_REQUESTED** [review]
Looks good, just 2 very minor things then we can merge this!

File: coder_sniffer/DrupalPractice/Test/General/OptionsTUnitTest.inc (hunk 2 above)
Comment (arkener) [reply coder_sniffer/DrupalPractice/Test/General/OptionsTUnitTest.inc]: Somehow I completely overlooked these coding standards issues on this file. Fixed now!

**Pull request merged**